### DISCUSSION TITLE
Delta sync time

### DISCUSSION THREAD

**Discussion by ll13j1209** [afs/rdf-delta#230] [Q&A] [answered]
Hi, when play around with rdf delta, I notice that currently, every 5 mins the delta server will try to sync the newest patch from the log server. Is there a way to config the time? Say I want to change the sync time to every 2 minutes. Thank you for your help!

**Comment by afs** [comment, marked as answer]
Not directly. It seems to be fixed in the code (`DeltaConnection`).

But there is a way to do this externally - send a lightweight query such as `ASK{}` every 2 minutes to the dataset. Each query request starts by checking the connection state.
